src/testUtils/java/org/broadinstitute/hellbender/testutils/IntegrationTestSpec.java

```diff
@@ -215,17 +234,20 @@ private void executeTest(String testName, CommandLineProgramTester testClass, St
     }
 
     public static void assertMatchingFiles(final List<File> resultFiles, final List<String> expectedFiles, final boolean compareBamFilesSorted, final ValidationStringency stringency) throws IOException {
-        assertMatchingFiles(resultFiles, expectedFiles, compareBamFilesSorted, stringency, false);
+        assertMatchingFiles(resultFiles, expectedFiles, compareBamFilesSorted, stringency, false, false);
     }
 
-    public static void assertMatchingFiles(final List<File> resultFiles, final List<String> expectedFiles, final boolean compareBamFilesSorted, final ValidationStringency stringency, final boolean trimWhiteSpace) throws IOException {
+    public static void assertMatchingFiles(final List<File> resultFiles, final List<String> expectedFiles, final boolean compareBamFilesSorted, final ValidationStringency stringency, final boolean trimWhiteSpace, final boolean skipIndexes) throws IOException {
         Assert.assertEquals(resultFiles.size(), expectedFiles.size());
         for (int i = 0; i < resultFiles.size(); i++) {
             final File resultFile = resultFiles.get(i);
             final String expectedFileName = expectedFiles.get(i);
             final File expectedFile = new File(expectedFileName);
             final boolean isIndex = INDEX_EXTENSIONS.stream().anyMatch(ext -> expectedFileName.endsWith(ext));
             if (isIndex) {
+                if (skipIndexes) {
+                    Assert.assertTrue(expectedFile.exists());
+                }
                 Assert.assertTrue(Files.equal(expectedFile, resultFile), "Resulting index file different from expected");
```

Comment:
Potential bug: we still compare the index file contents even if `skipIndexes` is true?

Reply:
@jamesemery

Reply:
whoops...